packages/components/src/Text/Text.tsx

```diff
@@ -2,6 +2,8 @@ import React, { PropsWithChildren } from "react";
 import { Typography, TypographyOptions } from "../Typography";
 
 interface TextProps {
+  maxLines?: "single" | "small" | "medium" | "large" | "larger";
```

Comment:
I read your comments about changing `base` to `medium` just waiting for some other feedback on this as `base` is a pretty common pattern. Will get back to you on this asap.

Reply:
What is the default for this prop if it is unset?

Reply:
@ZakaryH we have discussed this more internally and here are our thoughts:

1. We should try not to have an optional prop without a default. In this case, `maxLines` should be optional with a default of something like `unlimited` which will have no line clamp.

2. For the time being, let's go with a consistent `base` for the name of the option. We are going to discuss if `base` makes sense for this and other components going forward, as it may be a little bit confusing.